src/controllers/expenseController.js

```diff
@@ -0,0 +1,131 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  try {
+    const filteredExpenses = expensesService.getAllExpenses(
+      userId,
+      categories,
+      from,
+      to,
+    );
+
+    res.status(200).json(filteredExpenses);
+  } catch (error) {
+    res.status(500).json({ error: 'Internal server error' });
+  }
```

Comment:
It seems like you're catching exceptions but not doing much with them. Instead of just returning a generic 'Internal server error', it would be helpful to log the error message or even return it in the response (if it doesn't expose sensitive information). This would make debugging easier in case of an error.